celltype_annotation/scripts/cell-assign.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+## basic script to use for testing cellassign with
+# followed tutorial from scvi cellassign
+# https://docs.scvi-tools.org/en/stable/tutorials/notebooks/cellassign_tutorial.html
+# used Follicular lympyhoma reference on zenodo
+# https://zenodo.org/record/3372746#.ZDcGQ-zMJA0
+
+
+import os
+import anndata as adata
+import scvi
+from scvi.external import CellAssign
+import pandas as pd
+import numpy as np
+
+# read in public references as marker gene tables
+fl_file = os.path.join("..", "references", "FL_celltype_ensembl.csv")
+fl_mtx = pd.read_csv(fl_file, index_col=0)
+
+fl_noBcells_file = os.path.join("..", "references", "FL_celltype_ensembl-noBcells.csv")
+fl_noBcells_mtx = pd.read_csv(fl_noBcells_file, index_col=0)
+
+# file path to annotated sce
+anndata_dir = os.path.join("..", "data", "anndata")
+annotated_adata_file = os.path.join(anndata_dir, "SCPCL000295_anndata.hdf5")
+annotated_adata = adata.read_h5ad(annotated_adata_file)
+
+# index adata but use row data to do it
+subset_adata = annotated_adata[:, fl_mtx.index].copy()
+subset_adata.X = subset_adata.X.tocsr()
+
+# renormalize/ calcualate size factor on subset adata
```

Comment:
It looks like you are using the full data to calculate the size factors, but adding it to the subset, so I updated this comment. 

I wonder though if we can use the size factors we already calculated in R? Are those included in the `anndata` output somewhere? (Mostly a note for the future)
```suggestion
# add size factor to subset adata (calculated from full data)
```